django_tablib/datasets.py

```diff
@@ -20,7 +20,10 @@ def __init__(self, queryset, headers=None, encoding='utf-8'):
                 # django < 1.9
                 field_names = v_qs.field_names
             headers.extend(field_names)
-            headers.extend(v_qs.query.aggregate_select)
+            
+            if hasattr(v_qs.query, 'aggregate_select'):
+                # Django < 1.10
+                headers.extend(v_qs.query.aggregate_select)
```

Comment:
Well, it has been renamed [AFAICS](https://docs.djangoproject.com/en/2.0/releases/1.8/#aggregate-methods-and-modules). So we should still use it for >= 1.10, but with the name `annotation_select`.

Im not the owner of this repo, but it would be nice to get this PR in when updated.

Reply:
Thanks for the reminder :+1: